getting_started/scripting/gdscript/static_typing.rst

```diff
@@ -147,7 +147,7 @@ Variable casting
 ~~~~~~~~~~~~~~~~
 
 Type casting is a key concept in typed languages. We call the conversion
-of a value from from one type of another casting.
+of a value from one type of another casting.
```

Comment:
@cbscribe are you only quoting some of it, or do you want "casting" to be removed as well (if not I've resolved this).

Alternatively it could also be rephrased like this "Type casting is a key concept in typed languages. Which is the conversion of a value from one type of another."

Reply:
Your second idea is better, although that's an incomplete sentence, and still uses "of" incorrectly:

Type casting is a key concept in typed languages. Casting is the conversion of a value from one type to another.

Reply:
That "of" was a typo on my part, did indeed mean "to". Kinda funny, given this pull request is about fixing typos.

I've committed your suggestion.